pkg/issuer/acme/dns/azuredns/azuredns_test.go

```diff
@@ -288,8 +296,9 @@ func TestGetAuthorizationFederatedSPT(t *testing.T) {
 			}
 
 			w.Header().Set("Content-Type", "application/json")
-			accessToken := map[string]string{
+			accessToken := map[string]any{
 				"access_token": "abc",
+				"expires_in":   500,
```

Comment:
Is `500` some kind of magic value? Sorry if this is a dumb question. 🤔

Reply:
500 is a random number I picked. I had to provide some number since the Azure SDK requires this now.

Reply:
I see that Azure SDK mostly uses the magic number `3600`: https://github.com/search?q=repo%3AAzure%2Fazure-sdk-for-go%20expires_in&type=code. Should we do that too? It's a more "well-known" number, probably 1 hour?